impl/client/src/main/javascript/web/prompting/components/ScopedPentahoButtonComponent.js

```diff
@@ -57,16 +57,18 @@ define([ 'cdf/components/BaseComponent', 'common-ui/jquery-clean' ], function(Ba
      * @name ScopedPentahoButtonComponent#registerSubmitClickEvent
      * @private
      */
-    registerSubmitClickEvent : function() {
-      if (!this.viewReportButtonRegistered) {
+    registerSubmitClickEvent: function() {
+      if(!this.viewReportButtonRegistered) {
 
         var $container = $("#" + this.htmlObject).empty();
+        var disableButton = this.promptPanel.paramDefn.disableSubmitButton;
```

Comment:
paramDefn is available directly, so you can write `this.paramDefn.disableSubmitButton; `